### PR TITLE
EN-7433: storer to elastic - fix miniblock not found

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
- fixed a bug where a miniblock was not found because it was stored in the previous epoch's storage.
- removed the chain ID path from config/flag and fetched it from genesis nodes setup
- skip indexing intra-shard smart contract results (excepting metachain intra-shard)
- replaced the indexing with go routines with the serial one, because due to too many requests, some data wouldn't have been indexed.

Testing procedure is the same as for #2249

### CODE REVIEW

File: cmd/storer2elastic/dataprocessor/dataProcessor.go [hunk 1]
```diff
@@ -156,16 +156,18 @@ func (dp *dataProcessor) indexData(data *storer2ElasticData.HeaderData) error {
 	notarizedHeaders := dp.computeNotarizedHeaders(data.Header)
 	newBody := &block.Body{MiniBlocks: make([]*block.MiniBlock, 0)}
 	for _, mb := range data.Body.MiniBlocks {
-		if mb.Type == block.ReceiptBlock { // don't index receipt miniblocks
+		if mb.Type == block.ReceiptBlock ||
```
Comment (iulianpascalau) [inline cmd/storer2elastic/dataprocessor/dataProcessor.go:159]: maybe extract into a variable?

Comment (bogdan-rosianu) [reply cmd/storer2elastic/dataprocessor/dataProcessor.go]: done

File: cmd/storer2elastic/databasereader/databaseReader.go [hunk 2]
```diff
@@ -81,7 +81,9 @@ func (dr *databaseReader) GetDatabaseInfo() ([]*DatabaseInfo, error) {
 		epochStr := re.FindString(dirname)
 		epoch, errParseInt := strconv.ParseInt(epochStr, 10, 64)
 		if errParseInt != nil {
-			log.Warn("cannot parse epoch number from directory name", "directory name", dirname)
+			if dirname != "Static" {
```
Comment (iulianpascalau) [inline cmd/storer2elastic/databasereader/databaseReader.go:84]: magic string?

Comment (bogdan-rosianu) [reply cmd/storer2elastic/databasereader/databaseReader.go]: done. also in other places

**Review by iulianpascalau — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
👍

**Pull request merged**